MapleServer2/PacketHandlers/Game/MoveFieldHandler.cs

```diff
@@ -85,6 +85,7 @@ public static void HandleMove(GameSession session, PacketReader packet)
                     System.Console.WriteLine($"unknown portal type id: {srcPortal.PortalType}");
                     break;
             }
+            session.SendNotice($"type:{srcPortal.PortalType} target: {srcPortal.Target}, {srcPortal.TargetPortalId}");
```

Comment:
debug message?

Reply:
I thought I removed that.
I noticed that the portal type is really descriptive. PortaltType 0 is a normal field portal dungeonlobby to dungeon portal has portal type 1 and so on. So, figuring these out might be a nice way for the server to know what portal was used by the client.